### ISSUE TITLE
Audio Issues on mobile networks at the Beginning of Calls

### ISSUE THREAD

**Issue by 1cue-jrupp** [flutter-webrtc/flutter-webrtc#1354]
We're using flutter-webrtc together with dart-sip-ua for a softphone app in combination with an Asterisk server and OPUS. 

On really stable wifi networks everything works great. But in slightly unstable networks or mobile networks the audio quality at the beginning of calls is very bad. It stutters, pauses for a second, then plays slightly quicker to catch up multiple times. 

After some time (approx. 20 sec) the audio quality becomes great and works fine.

- Am I right that flutter-webrtc optimizes things like audio bandwidth or jitter buffer in the first seconds to get best quality?
- Is there any way to make the call quality good from the beginning of each call?
- For example, is it possible to let the audio session start with the lowest quality setting and only try to enhance if network is good enough?

We can reproduce this on Android and iOS, it seems not to be related to Flutter or Plugin Versions as it is reproducable with different versions.

**Comment by reduxdj** [comment]
Yes, on IOS especially on outbound calls, there's 30 seconds of jitter.
